src/types.ts

```diff
@@ -99,3 +99,8 @@ export type TaskGroupResult = {
   groupId: string;
   map: ResultMap;
 };
+
+export enum AllowedGroup {
```

Comment:
I personally think this should just be a type rather than an enum:

`type Groups = 'client' | 'service'`. Then you don't need to import the enum from the package